### PR TITLE
add catalog.rb code for browse

### PR DESCRIPTION
**Pull request by gordonleacock**
Browse code for roger to examine.

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -236,9 +236,13 @@ def get_date_options
...
   def get_decade_browse_options
-    decades = ["Any Decade", 1890, 1900, 1910, 1920, 1930, 1940, 1950, 1960, 1970, 1980, 1990, 2000, 2010, 2010]
+    decades = [1890, 1900, 1910, 1920, 1930, 1940, 1950, 1960, 1970, 1980, 1990, 2000, 2010, 2010]
     decades_range = (1..decades.count)
     decade_options = [];
+
+    item = ["Any Decade", "Any Decade"]
+    decade_options.push item
+
     decades_range.each do |m|
```
Comment (respinos) [inline app/helpers/application_helper.rb]: Could this just be `decade.each do |m|`?

File: app/views/catalog/browse.html.erb [hunk 2]
```diff
@@ -1,28 +1,54 @@
...
+    end
+    if (params['date_issued_yyyy_ti'] != "Any Year")
+        this_decade = "" 
+    end
+        %>
+    <h3>Browsing: <%= params['date_issued_dd_ti'] %> of <%= this_month %>, <%= params['date_issued_yyyy_ti'] %><%= this_decade %></h3>  
+
+    <%= render 'search_results' %>
+
+      <% if (false) %>
```
Comment (respinos) [inline app/views/catalog/browse.html.erb]: This block be deleted now, right?

File: app/helpers/blacklight_helper.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+module BlacklightHelper
+  include Blacklight::BlacklightHelperBehavior
+
+  def presenter(document)
+    STDERR.puts "AHOY AHOY AHOY #{action_name}"
```
Comment (respinos) [inline app/helpers/blacklight_helper.rb]: This ridiculous line can be deleted. Sigh.

File: app/views/catalog/browse.html.erb (hunk 2 above)
Comment (gordonleacock) [inline app/views/catalog/browse.html.erb]: Nope, that's how I create the custom heading at the top of  the view.

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (gordonleacock) [inline app/helpers/application_helper.rb]: Range isn't happy with a character value like "Any Decade".  ;-(

File: app/helpers/blacklight_helper.rb (hunk 3 above)
Comment (gordonleacock) [inline app/helpers/blacklight_helper.rb]: Done.

**Pull request merged**